lib/mix/tasks/surface/surface.init/project_patchers/formatter.ex

```diff
@@ -57,7 +57,7 @@ defmodule Mix.Tasks.Surface.Init.ProjectPatchers.Formatter do
   def add_surface_inputs_to_formatter_config() do
     %{
       name: "Add file extensions to :surface_inputs",
-      patch: &FilePatchers.Formatter.add_config(&1, :surface_inputs, ~S(["{lib,test}/**/*.{ex,exs,sface}"])),
+      patch: &FilePatchers.Formatter.add_config(&1, :surface_inputs, ~S(["{lib,test,priv}/**/*.{ex,exs,sface}"])),
```

Comment:
Would something like `{lib,test,priv/catalogue}/**/*.{ex,exs,sface}` work? Since we may have other folders in `priv` that we don't want to touch by default.